core/src/main/java/bisq/core/dao/node/parser/TxOutputParser.java

```diff
@@ -115,6 +115,8 @@ void processTxOutput(TempTxOutput tempTxOutput) {
                 // The LOCKUP BSQ is burnt unless the output exactly matches the input, that would cause the
                 // output to not be BSQ output at all
                 handleUnlockBondTx(tempTxOutput);
+            } else if (isBurnFeeTxOutput(tempTxOutput)) {
```

Comment:
I found this a bit confusing. Seems correct but might help with extra comment or moving this into the next else if, eg.
`else if (!isBurnFeeTxOutput(tempTxOutput) && availableInputValue > 0 && availableInputValue >= txOutputValue) {`

Maybe change function name to `isBurnFeeBtcTxOutput`

Reply:
Yes, true. Will try to make it more clear.